webui/react/src/pages/F_ExpList/F_ExperimentList.tsx

```diff
@@ -81,6 +98,21 @@ const F_ExperimentList: React.FC<Props> = ({ project }) => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [page, sortString]);
 
+  useEffect(() => {
+    // useSettings load the default value first, and then load the data from DB
+    // use this useEffect to re-init the correct useSettings value when settings.filterset is changed
+    let ignore = false;
+
+    if (!ignore) {
+      const formSet: FilterFormSet = JSON.parse(settings.filterset);
```

Comment:
Should we use an io-ts parser here to ensure the input is what we expect?

Reply:
does useSettings store object (i know it stores array)? IIRC, it doesnt store objects

Reply:
I don't think so -- you might not be able to use the settings pattern we use elsewhere, but you should be able to pass the result of the json parse into the parser?